packages/cli-e2e/utils/misc.ts

```diff
@@ -0,0 +1,56 @@
+import {ProcessManager} from './processManager';
```

Comment:
Rename the file git.ts

`misc` it's a large net :P

I know us developer, we're a lazy bunch and we'll end up dropping everything in there 😂  (or maybe it's just me 🤷 )